uber/models.py

```diff
@@ -1167,7 +1165,7 @@ def _staffing_adjustments(self):
         if self.ribbon == c.DEPT_HEAD_RIBBON:
             self.staffing = True
             self.trusted_depts = self.assigned_depts
-            if c.SHIFT_CUSTOM_BADGES or c.STAFF_BADGE not in c.PREASSIGNED_BADGE_TYPES:
+            if (c.SHIFT_CUSTOM_BADGES or c.BEFORE_PRINTED_BADGE_DEADLINE) or c.STAFF_BADGE not in c.PREASSIGNED_BADGE_TYPES:
```

Comment:
Just for my own clarification, what determines the value of `c.SHIFT_CUSTOM_BADGES`?